src/redux/ui/screens/input-data/actions.ts

```diff
@@ -24,12 +33,12 @@ export const toggleValidatorMultiple = (
     });
 };
 
-export const selectInput = (screenKey: string, inputData: any, inputKey: string) => async (
+export const setScreenInputData = (screenKey: string, inputData: any, inputKey: string) => async (
```

Comment:
in some cases we may need to add more fields at once, you can merge `inputData: any, inputKey: string` into an object:
```ts
{
  [key: string]: any
}
```